<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d1_Screen_7</class>
 <widget class="QWidget" name="d1_Screen_7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="d1_7_widgetBackground" native="true">
     <property name="styleSheet">
      <string notr="true">#d1_7_widgetBackground{
	border-image: url(:/d1_popupFon.png);
}</string>
     </property>
     <widget class="QLabel" name="d1_7_lblInfo">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>124</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Baloo Cyrillic</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#d1_7_lblInfo{
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Выдача игрушки</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="d1_7_lblRainbow">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>271</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#d1_7_lblRainbow{
	border-image: url(:/d1_rainbow2.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="d1_7_lblRedHorse">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>330</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#d1_7_lblRedHorse{
	border-image: url(:/d1_redHorse.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
